skaffold/README.md

```diff
@@ -0,0 +1,56 @@
+# Skaffold files
+
+Demonstation files for using [skaffold](https://github.com/GoogleContainerTools/skaffold) to develop configurations.
+
+## Purpose
+
+These samples provide developers a faster way to iterate on configurations for the ForgeRock platform. The idea is to perform iteration using skaffold, and then when you are happy with the configuration, publish it to git.  This shortens the development cycle from this:
+
+`edit configuration -> git commit -> git push -> create PR -> merge PR -> redeploy helm chart `
+
+To this:
+
+`edit configuration`
+
+With skaffold handling the redeployment process.
+
+## Usage
+
+Each folder contains a child Dockerfile that is based on the product Dockerfile (for example, for ig the child is derived `FROM gcr.io/engineering-devops/ig` ).
+
+The configuration for the product is copied into this child image (COPY *). This results in a docker image that has the configuration baked into the image, instead of being pulled from git (as is the case in production).
```

Comment:
I guess the * after Copy * is making everything beyond this italicized, please fix this.